plugins/pencilblue/controllers/api/content/article_api_controller.js

```diff
@@ -41,26 +42,52 @@ module.exports = function(pb) {
     ArticleApiController.prototype.init = function(context, cb) {
         var self = this;
         var init = function(err) {
-            
+
             /**
-             * 
+             *
              * @property service
              * @type {ArticleServiceV2}
              */
             self.service = new ArticleServiceV2(self.getServiceContext());
-            
+
             /**
              *
              * @property commentService
              * @type {CommentService}
              */
             self.commentService = new CommentService(self.getServiceContext());
-                
+
             cb(err, true);
         };
         ArticleApiController.super_.prototype.init.apply(this, [context, init]);
     };
-    
+
+    /**
+     * Retrieves one or more resources from a collection.
+     * @method getAll
+     * @param {Function} cb
+     */
+    ArticleApiController.prototype.getAll = function(cb) {
+        var self = this;
+        var options = this.processQuery();
+
+        this.service.getAllWithCount(options, function(err, obj) {
+            if (util.isError(err)) {
+                return cb(err);
+            }
+            else if (util.isNullOrUndefined(obj)) {
+                return self.notFound(cb);
+            }
+
+            var userService = new UserService(self.getServiceContext());
+            userService.getAuthors(obj.data, function(err, articlesWithAuthorNames) {
```

Comment:
This function along with the whole prototype ```UserService``` has been deprecated and replaced by ```UserServiceV2```.  Is this so we get the author information on the article list?